core/src/main/java/com/scalar/db/api/OperationBuilder.java

```diff
@@ -416,20 +417,28 @@ interface WhereAnd<T> {
     /**
      * Appends the specified set of or-wise conditions.
      *
-     * @param conditions a set of conditions
+     * @param orConditionSet a set of or-wise conditions
      * @return the operation builder
      */
-    T where(ScanBuilder.OrConditionSet conditions);
+    T where(ScanBuilder.OrConditionSet orConditionSet);
```

Comment:
BTW, can we also add `where(Set<ScanBuilder.OrConditionSet > andConditionSets)` to `WhereAnd<T>` as well, it could cause signature conflicts though?

Reply:
Unfortunately, we cannot add it since it causes the signature conflict, as you guess.

Reply:
Okay. In that case, how about changing the method name to address the signature conflict as follows?

```java
  interface WhereAnd<T> {
    T where(ScanBuilder.OrConditionSet orConditionSet);

    T whereAnd(Set<ScanBuilder.OrConditionSet> orConditionSets);
  }

  interface WhereOr<T> {
    T where(ScanBuilder.AndConditionSet andConditionSet);

    T whereOr(Set<ScanBuilder.AndConditionSet> andConditionSets);
  }
```

Reply:
Thanks for the solution. Umm, it's a bit unstylish, but right, consistency is more important here, I think.

Reply:
Fixed in 1b9ae6a. PTAL when you have time!